computers/paratera-th2/paratera-th2-configure.yml

```diff
@@ -0,0 +1,7 @@
+---
+port:   2200
+look_for_keys:          True
+key_filename:           '/home/unkcpz/.ssh/papp-test.id'
```

Comment:
I guess these two settings should be configured by the users themselves?

Reply:
One question here, `look_for_keys` must be set to `True` if `key_filename` is specified here and the filename is not the default `id_rsa`, correct?

Reply:
From to the [docs](https://aiida.readthedocs.io/projects/aiida-core/en/latest/_modules/aiida/transports/plugins/ssh.html?highlight=look_for_keys), I understand it is needed only when you don't specify the path to the key file.
In any case, it is true by default.